libs/postgres_ffi/src/lib.rs

```diff
@@ -204,6 +206,8 @@ pub fn prepare_initdb_command(
     target_dir: &Path,
     user: &str,
 ) -> anyhow::Result<Command> {
+    let initdb_bin = pg_bin_dir.join("initdb");
+    info!("running {initdb_bin:?} in {target_dir:?}, libdir: {pg_lib_dir:?}");
```

Comment:
Nice!

So, does env var `RUST_LOG` contain  `postgres_ffi=info` in all relevant environments? (unit tests, python tests, staging, prod)?

Reply:
AFAIK, it's not needed since we have `info` there that should handle this enabling either.
> unit tests

Nothing there, but neighbour https://github.com/neondatabase/neon/pull/3284 should enable `info` there too.